inspirehep/records/marshmallow/literature/latex.py

```diff
@@ -17,7 +17,8 @@
 class LatexSchema(Schema):
     arxiv_eprints = fields.Raw()
     authors = fields.Method("get_author_names")
-    citations = fields.Method("get_citations", default=0)
+    # It has to be string, otherwise if result is 0 then it's not rendered in latex
```

Comment:
I think that's what the `default` was for.

Reply:
Still with default 0 it was returning `int` 0 and because of this whole line from latex was not added to the output and was failing tests.

Reply:
``default='0'`` :)

Reply:
Still adding default '0' won't help anyhow, as this default value will be added only when value won't be provided, and not when value will be 0. Actually default for this field is redundant, as there should be always some value. Only chance for not having value is when citation query count will throw exception.